web/server/vue-cli/src/views/Reports.vue

```diff
@@ -140,7 +144,17 @@
           </v-chip>
         </template>
 
-        <template #item.reviewData="{ item }">
+
+        <template v-if="reportFilter.isUnique" #item.reviewData="{ item }">
+          <span v-if="sameReportsCounter != reports.length" />
+          <review-status-icon
```

Comment:
It would be good to add some padding to these icons. You can do it by adding this class to this item: `class="pa-1"`